pycbc/psd/__init__.py

```diff
@@ -27,7 +28,7 @@
 from pycbc.types import ensure_one_opt, ensure_one_opt_multi_ifo
 
 def from_cli(opt, length, delta_f, low_frequency_cutoff,
-             strain=None, dyn_range_factor=1, precision=None):
+             strain=None, dyn_range_factor=1, precision=None, **kwargs):
```

Comment:
Should this just get the option from opt like other things do?  I think in general it is better to be explicit, but in this case, we've somewhat set a convention and it would be better to not mix conventions.

Reply:
I think the difference here with things like length, df, and flow is that those are required for any psd. The options and extra kwargs are more like the choice of psd type which we have already set up to take from the command line. Since you are adding the option to the command line anyway, it would probably be most consistent to use the options directly.

Reply:
@ahnitz Thanks for this suggestion! I just realized that our extra options are included in `opt` if I understand it correctly.